src/status_im2/common/resources.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im2.common.resources)
+
+(def ui
+  {:add-new-contact (js/require "../resources/images/ui2/add-contact.png")
+   :lifestyle       (js/require "../resources/images/ui2/lifestyle.png")
+   :music           (js/require "../resources/images/ui2/music.png")
+   :podcasts        (js/require "../resources/images/ui2/podcasts.png")
+   :intro-1         (js/require "../resources/images/ui2/intro-1.png")
```

Comment:
The images added contain a transparent padding top (i.e. some transparent pixels), is it correct?
 I've seen this sometimes in the designs, but I'm not sure whether that's due to the way figma exports the files or it's intended by the design team.

Reply:
yes, let's check with design team about this 👍 I exported using Figma which is the method we normally use afaik

Reply:
images look good in the app btw 👍

Reply:
yes they looks different images
![image](https://user-images.githubusercontent.com/17097240/222508191-33edc35f-bdc6-4819-ab76-006ee82acb5b.png)

Fresh Install flow uses different image then app locked etc.
(probably they are just placeholder and going to be replaced, not sure)

Reply:
@J-Son89 @Parveshdhull I'm talking about these transparent pixels:
![image](https://user-images.githubusercontent.com/90291778/222523355-76bb2dbb-d212-4c97-9f17-9ce1b324c7c0.png)

Is it ok to keep that "padding-like"?

Reply:
I think we should use this kind of images without transparent pixels, and add paddings programatically if necessary

Reply:
I'll discuss with design team!